lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -80,11 +80,24 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
-
         self._set_log_level(options)
+        self.enqueue_all_shuffled_tasks(options)
 
-        for course_key in self._get_course_keys(options):
-            self.enqueue_compute_grades_for_course_tasks(course_key, options)
+        #for course_key in self._get_course_keys(options):
```

Comment:
Yes!  And then some!  The whole enqueue_compute_grades_for_course_tasks method can go now.